utils/environment.yml

```diff
@@ -1,78 +1,85 @@
+# To recreate this file:
+# conda create -n conda_docs -c conda-forge \
+# sphinx \
+# sphinx-autobuild \
+# sphinx-argparse==0.2.5 \
```

Comment:
This change to environment.yaml should be tested on read the docs before merging